tests/cpu_efm32_features/main.c

```diff
@@ -26,21 +26,7 @@
 
 int main(void)
 {
-    /* test if uart_config[i].mode is set to a know value */
-    for (unsigned i = 0; i < UART_NUMOF; i++) {
-        printf("UART %u mode: ", i);
-
-        switch (uart_config[i].mode) {
-            case UART_MODE_8N1:
-                puts("8N1");
-                break;
-            case UART_MODE_8E1:
-                puts("8E1");
-                break;
-            default:
-                puts("unknown");
-        }
-    }
+    puts("Board booted, with some EFM32 features enabled or disabled.");
```

Comment:
As this test is a NOP now, why not just remove it?

Reply:
Well, it's not exactly a NOP. The Makefile disables the LEUART peripheral, and this test checks that it actually compiles. Without this test, there is no test/code path active that validates this.